### DISCUSSION TITLE
Why did the Screensaver shortcut disappear?

### DISCUSSION THREAD

**Discussion by jbcurran** [hrvach/deskhop#173] [Q&A]
I'd like to continue to upgrade to the newest releases, but as far as I can tell in the code, the Screensaver shortcut was removed awhile back. I toggle the screensaver often throughout the day, so having a shortcut is much faster than opening the config. Thanks!

**Comment by hrvach** [comment]
We can bring it back, sure. Since you're using it actively, maybe you can help with a suggestion. Would it be more convenient to have a dedicated ON and OFF shortcuts, or a toggle shortcut is enough ?

Two different shortcuts makes it more difficult for me to remember them, while a toggle one leaves me never sure about what's the current system state. Opinions?

**Comment by jbcurran** [comment]
Awesome. It would be more convent to have a dedicated ON and OFF shortcut, as sometimes I don't know what the state is and just have to wait. Usually this means coming back later in the night to see my cursor bouncing around the lock screen. 

In my perfect world I would have:

- right shift + f12 + a to toggle the a screen saver
- right shift +f12 + b to toggle the b screen saver
- ability to include 2 new LEDs to the picos to indicate whether a screensaver is on for each screen.
     - I tried to implement this awhile back during the v0.5X days, but unfortunately my C skills are not up to the task.
     
In reality:
I'll take the right shift + f12 + s shortcut back so that I can upgrade to v0.66
OR
- right shift + f12 + s to turn screensavers on
- right shift + f12 + x to turn screensavers off (I choose x because its below the s, so like a on/off switch)

I really appreciate all the work you've put into this! Thank you!

**Reply by hrvach** [reply]
Please check prerelease 0.68 and let me know if it works OK for you. it's a tad different - left ctrl + right shift + s or x as you suggested, like a switch.

**Comment by hrvach** [comment]
Closing for the time being, please be so kind to reopen if not fixed.
